services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala

```diff
@@ -1,11 +1,16 @@
 package cromwell.pipeline.service
 
+import cromwell.pipeline.datastorage.dao.repository.ProjectRepository
 import cromwell.pipeline.datastorage.dto._
 import cromwell.pipeline.womtool.WomToolAPI
 
 import scala.concurrent.{ ExecutionContext, Future }
 
-class ProjectFileService(womTool: WomToolAPI, projectVersioning: ProjectVersioning[VersioningException])(
+class ProjectFileService(
+  projectRepository: ProjectRepository,
```

Comment:
Only `ProjectService` should use `ProjectRepository`. If you need to fetch a project you should use `ProjectService` instead of `ProjectRepository`